Dockerfile

```diff
@@ -23,8 +23,15 @@ RUN git clone https://github.com/norwoodj/helm-docs -b v1.11.0 \
     && go build ./cmd/helm-docs \
     && cp helm-docs /go/bin/helm-docs
 
+RUN git clone https://github.com/ubergesundheit/helm-schema \
```

Comment:
It'll make sense to update this to pick the upstream repo + target sha once all the features are merged upstream.

Reply:
Updated in https://github.com/cilium/helm-toolbox/pull/3/commits/826a209f0112f76d9b18000c59fc1979a869d876